src/include/wiredtiger.in

```diff
@@ -723,7 +723,7 @@ struct __wt_cursor {
 #define	WT_CURSTD_DUMP_PRINT		0x0000800u
 #define	WT_CURSTD_HS_READ_ALL	0x0001000u
 #define	WT_CURSTD_HS_READ_COMMITTED	0x0002000u
-#define	WT_CURSTD_IGNORE_TOMBSTONE	0x0004000u
+#define	WT_CURSTD_RETURN_TOMBSTONE	0x0004000u
```

Comment:
We usually have our flags in alphabetically order. We have a script flags.py which takes care of automatically generating the flag values, and it will contain instructions on how to add a new flag.